OpenSim/Examples/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -173,8 +173,27 @@ void addDeviceConsoleReporterToModel(Model& model, Device& device,
 // the exercise. The project should execute without making any changes (you
 // should see the unassisted hopper hop slightly).
 //------------------------------------------------------------------------------
-int main()
-{
+int main(int argc, char* argv[]) {
+    //==========================================================================
+    // Command line argument parsing.
+    //==========================================================================
+    // Following are arguments to this executable:
+    // (1) 'noVisualizer' -- [Optional] Do not show visualizer. Default
+    //                       behavior is to show visualizer.
+    // (2) 'simulateOnce' -- [Optional] Simulate once and exit the program.
+    //                       Default behavior is to simulate multiple times
+    //                       until user requests to stop.
+    
+    // Suppress/show visualizer.
+    bool showVisualizer{true};
+    // Simulate just once or until user stops.
+    bool simulateOnce{false};
+    for(int i = 0; i < argc; ++i)
+        if(strcmp(argv[i], "noVisualizer") == 0)
```

Comment:
Quote is not required when calling the executable. I had to emphasize the argument.
I have removed the quote now.